packages/components/message/src/message.vue

```diff
@@ -80,7 +78,7 @@ export default defineComponent({
 
     const typeClass = computed(() => {
       const type = props.type
-      return type && TypeComponentsMap[type] ? `el-message-icon--${type}` : ''
+      return type && TypeComponentsMap[type] ? ns.bm('icon', type) : ''
```

Comment:
Same here

Reply:
Thanks for pointing it out, it has been fixed